### PR TITLE
Add information on withoutMiddleware and rate limit throttling

### PR DESCRIPTION
**Pull request by josephcrowell**
Adds some additional information on middleware and configuring api rate limits

### CODE REVIEW

File: services/router.md [hunk 1]
```diff
@@ -203,16 +203,50 @@ Route::group(['middleware' => 'Path\To\Your\Middleware'], function() {
...
 ```php
 Route::middleware(['Path\To\Your\Middleware'])->group(function() {
     Route::get('info', 'Acme\News@info');
 });
 ```
 
+And finally, if you want to remove middleware from a route you can it like this
+
+```php
+Route::withoutMiddleware("Path\To\Removed\Middleware")
```
Comment (LukeTowers) [inline services/router.md]:
```suggestion
Route::withoutMiddleware(\Path\To\Removed\Middleware::class)
```

File: services/router.md [hunk 2]
```diff
@@ -203,16 +203,50 @@ Route::group(['middleware' => 'Path\To\Your\Middleware'], function() {
...
+Route::withoutMiddleware(\Path\To\Removed\Middleware::class)
+    ->get('info', 'Acme\News@info');
+```
+
 You can of course add more than one middleware in a group, just one is used in the above examples for convenience.
 
+## Throttle middleware
+
+When using Winter's API middleware, you can remove the default 60 request per minute rate throttle applied to Winter's
+API middleware and replace it with your own. For example
```
Comment (LukeTowers) [inline services/router.md]:
```suggestion
When using Winter's `api` middleware, you can remove the default rate limiting of 60 requests per minute and replace it with your own by using the below example:
```

File: services/router.md (hunk 1 above)
Comment (josephcrowell) [reply services/router.md]: Nice. I'll try making that change in my code.

Comment (LukeTowers) [reply services/router.md]: It's a lot nicer for IDEs so I'm trying to gradually shift all the core code and examples in that direction.

**Comment by mjauvin** [comment]
There are missing words in this PR...

e.g. `... you can it like this`

**Pull request merged**